BringgTrackingFramework/fastlane/Appfile

```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
```

Comment:
This looks like all lines are commented out
consider removing this file

Reply:
done